aws-ts-eks-hello-world/index.ts

```diff
@@ -80,6 +91,20 @@ const service = new k8s.core.v1.Service(name,
     }
 );
 
+let dnsRecord = undefined;
+if (hostedZoneName != undefined) {
+    const zoneId = aws.route53.getZone({ name: hostedZoneName }).then(z => z.id);
+    const record = new aws.route53.Record(name, {
+        zoneId: zoneId,
+        type: "CNAME",
+        ttl: 300,
+        name: service.metadata.name,
+        records: [service.status.loadBalancer.ingress[0].hostname],
+    });
+    dnsRecord = pulumi.interpolate`http://${record.fqdn}/`;
+}
+
 // Export the Service name and public LoadBalancer Endpoint
 export const serviceName = service.metadata.name;
 export const serviceHostname = service.status.apply(s => s.loadBalancer.ingress[0].hostname)
+export const serviceDnsName = dnsRecord;
```

Comment:
Would it make more sense to move this inside the hostedZone conditional? Looks like it will just be undefined otherwise.